v2/azure-arm.yaml

```diff
@@ -13,8 +13,10 @@ typesOutputPath: api
 # containing the mod file above.
 typeRegistrationOutputFile: internal/controllers/controller_resources_gen.go
 
-# samplesUrl is the base URL used for linking to samples when we generate a list of supported resources
-samplesUrl: https://github.com/Azure/azure-service-operator/tree/main/v2/config/samples
+# rootUrl is the root URL for ASOv2 repo
+rootUrl: https://github.com/Azure/azure-service-operator/tree/main/v2
+# samplesPath is the base Path used for walking through to samples when we generate a list of supported resources
```

Comment:
You should document what folder the relative path is resolved from - I think it's the folder containing this config YAML file, but am I right?